src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/admin/AdminPage.java

```diff
@@ -28,14 +29,42 @@ public static AdminPage initPage(WebDriver driver) {
 	}
 
 	/**
-	 * Compares the expected page title to the actual page title.
+	 * Navigate to the admin panel page on path 'admin/'.
+	 *
+	 * @param driver	{@Link WebDriver}
+	 */
+	public void goToAdminPage(WebDriver driver) {
+		get(driver, ADMIN_RELATIVE_URL);
+	}
+
+	/**
+	 *  Determines if admin panel title is
+	 *  visible on the admin panel page.
 	 *
 	 * @return {@link Boolean}
 	 */
-	public boolean comparePageTitle(String pageTitle) {
-		int titleFound = body.findElements(By.xpath("//h1[contains(text(),'" + pageTitle + "')]"))
-				.size();
+	public boolean adminPanelTitleVisible() {
+		return driver.findElements(By.id("t-admin-panel-title"))
+				.size() == 1;
+	}
 
-		return titleFound > 0;
+	/**
+	 *  Determines if admin panel button is
+	 *  visible on the navbar.
+	 *
+	 * @return {@link Boolean}
+	 */
+	public boolean adminPanelButtonVisible() {
+		return driver.findElements(By.id("t-admin-panel-btn"))
+				.size() == 1;
+	}
+
+	/**
+	 *  Clicks on the admin panel button to navigate
+	 *  to the admin panel page.
+	 */
+	public void clickAdminButton() {
+		adminPanelBtn.click();
+		waitForTime(500);
```

Comment:
I am sure you see this `waitForTime` sprinkled all over, it is a terrible, horrible bad thing that I created and am trying to get rid of.  The reason it is bad, is no matter what happens it must wait for 500ms. One of these is not bad, but have them everywhere really slows down the tests.  A better way here would be:

```
WebDriverWait wait = new WebDriverWait(driver, 2);
wait.until(ExpectedConditions.urlContains("/admin"));

Reply:
Okay sounds good, I will add that instead